larray/core/session.py

```diff
@@ -401,6 +637,19 @@ def summary(self, template=None):
 
 
 def local_arrays(depth=0):
+    """
+    Returns a session containing all local arrays.
```

Comment:
return Session((k, d[k]) for k in sorted(d.keys()) if isinstance(d[k], LArray))  

--> isinstance(d[k], LArray)